trace/widgets/axis_settings.py

```diff
@@ -41,6 +44,13 @@ def __init__(self, parent: QWidget, plot: PyDMArchiverTimePlot, axis: BasePlotAx
         y_grid_row = SettingsRowItem(self, "Y Axis Gridline", self.grid_checkbox)
         main_layout.addLayout(y_grid_row)
 
+        self.palette_modal = CurveColorPaletteModal(self)
+        self.curve_palette_button = QPushButton("Select")
+        self.curve_palette_button.clicked.connect(self.palette_modal.show)
+        self.palette_modal.sig_palette_changed.connect(self.sig_curve_palette_changed.emit)
+        palette_row = SettingsRowItem(self, "  Curve Palette", self.curve_palette_button)
```

Comment:
```suggestion
        palette_row = SettingsRowItem(self, "Curve Palette", self.curve_palette_button)
```

Doesn't need the indent here. The image below is what it looks like with the indent.

<img width="200" height="147" alt="Image" src="https://github.com/user-attachments/assets/7f5f56a4-fb99-416c-8543-11f2b7629354" />